examples/hello_services/HelloServices/main.m

```diff
@@ -0,0 +1,14 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <UIKit/UIKit.h>
+#import <Flutter/Flutter.h>
+
+int main(int argc, char * argv[]) {
+    FlutterInit(argc, (const char**)argv);
```

Comment:
We can use `-[NSProcessInfo arguments]` to initialize Flutter in the FlutterViewController, if we did that, the user would not have to make this call. Maybe that is a better way to go (in a future patch).